hosts/web01/hardware-configuration.nix

```diff
@@ -0,0 +1,77 @@
+# Do not modify this file!  It was generated by ‘nixos-generate-config’
+# and may be overwritten by future invocations.  Please make changes
+# to /etc/nixos/configuration.nix instead.
+{ modulesPath
+, ...
+}: {
+  imports = [
+    (modulesPath + "/profiles/qemu-guest.nix")
+  ];
+
+  boot.initrd.availableKernelModules = [ "ata_piix" "virtio_pci" "xhci_pci" "sd_mod" "sr_mod" ];
+  boot.initrd.kernelModules = [ ];
+  boot.kernelModules = [ ];
+  boot.extraModulePackages = [ ];
+
+  disko.devices = {
+    disk = {
+      sda = {
+        type = "disk";
+        device = "/dev/sda";
+        content = {
+          type = "table";
+          format = "gpt";
+          partitions = [
```

Comment:
In case it's not too late, I can recommend switching to the new gpt type:

https://github.com/nix-community/disko/blob/15c4d57b41b6b57024aec015e5d30a4ed4713034/example/simple-efi.nix#L10

Advantage is that it uses gpt labels to mount disks